diff --git a/src/public_html/index.php b/src/public_html/index.php
index a684a41..96d3cee 100644
--- a/src/public_html/index.php
+++ b/src/public_html/index.php
@@ -1,5 +1,5 @@
 <?php
 require_once(dirname(__FILE__).'/../config/ProjectConfiguration.class.php');
 
-$configuration = ProjectConfiguration::getApplicationConfiguration('frontend', 'prod', false);
+$configuration = ProjectConfiguration::getApplicationConfiguration('frontend', 'dev', true);
 sfContext::createInstance($configuration)->dispatch();
